test/fixtures/templates-jade.js

```diff
@@ -0,0 +1,85 @@
+// TEST_1
+@Component({
+  selector: 'app'
+})
+@View({
+  templateUrl: './app.jade',
+  styleUrls: ['./app.css'],
+  directives: [CORE_DIRECTIVES]
+})
+export class App {}
+
+// TEST_2
+@Component({
+  selector: 'app'
+})
+@View({
+  templateUrl: './app.jade',
+  styleUrls: [
+    './app.css'
+  ]
+})
+export class App {}
+
+// TEST_3
+@Component({
+  selector: 'app'
+})
+@View({
+  templateUrl: './app.jade',
+  styleUrls: [
+    './app.css',
+    './common.css'
+  ],
+  directives: [CORE_DIRECTIVES]
+})
+export class App {}
+
+// TEST_4
+@Component({
+  selector: 'app'
+})
+@View({
+  templateUrl: './app.jade',
+  styleUrls: [
+    './app.css',
+    './common.css',
+    './app.css',
+    './common.css'
+  ],
+  directives: [CORE_DIRECTIVES]
+})
+export class App {}
+
+// TEST_5
+@Component({
+  selector: 'app'
+})
+@View({
+  templateUrl: './app.jade',
+  styleUrls: ['./app.css', './common.css'],
+  directives: [CORE_DIRECTIVES]
+})
+export class App {}
+
+// TEST_6
+@Component({
+  selector: 'app'
+})
+@View({ templateUrl: './app.jade', styleUrls: ['./app.css', './common.css'] })
+export class App {}
+
+// TEST_7
+@Component({
+  selector: 'app'
+})
+@View({ templateUrl: './app.jade', styleUrls: ['./app.css', './common.css'], directives: [CORE_DIRECTIVES] })
+export class App {}
+
+// TEST_8
+@Component({
+  selector: 'app'
+})
+@View({ templateUrl: './app.jade',
+  styleUrls: ['./app.css', './common.css'], directives: [CORE_DIRECTIVES] })
+export class App {}
```

Comment:
Thought they were not needed for the jade part as they run identically on the html part (I guessed they were sanity checks for components that need no including process)